Makefile

```diff
@@ -176,6 +177,13 @@ tests/%.parse: tests/%
 tests/%.prove: tests/%
 	$(TEST) prove --backend $(TEST_SYMBOLIC_BACKEND) $< --format-failures --def-module $(KPROVE_MODULE)
 
+tests/%.cannot-prove: tests/%
+	@echo "$<"
+	@failed=0 \
+	  ; $(TEST) prove --backend $(TEST_SYMBOLIC_BACKEND) $< --format-failures --def-module $(KPROVE_MODULE) --boundary-cells k 2> /dev/null \
+	  || failed=1 \
+	  ; test $$failed = 1
```

Comment:
Let's instead test failed tests via an expected output file, with the failed proof state.

EG:

```
$(TEST) prove --backend $(TEST_SYMBOLIC_BACKEND) $< --format-failures --def-module $(KPROVE_MODULE) --boundary-cells k > $<.out \
    || $(CHECK) $<.expected $<.out
```

Then add expected output files (by running the test then copying the expected output file into place).

Reply:
If `$(CHECK)` isn't defined yet (not sure), it should be `CHECK := git --no-pager diff --no-index`

Reply:
Done